themes/theme-default/src/client/components/NavbarBrand.vue

```diff
@@ -24,12 +24,20 @@ const navbarBrandLogo = computed(() => {
   }
   return themeLocale.value.logo
 })
+const navbarBrandLogoAlt = computed(
+  () => themeLocale.value.logoAlt ?? navbarBrandTitle.value,
+)
+const navBarLogoAltMatchesTitle = computed(
+  () =>
+    navbarBrandTitle.value.toLocaleUpperCase().trim() ===
+    navbarBrandLogoAlt.value.toLocaleUpperCase().trim(),
+)
 const NavbarBrandLogo: FunctionalComponent = () => {
   if (!navbarBrandLogo.value) return null
   const img = h('img', {
     class: 'logo',
     src: withBase(navbarBrandLogo.value),
-    alt: '',
+    alt: navbarBrandLogoAlt.value,
```

Comment:
I still think reading out the title could be more natural, remove the aria-hidden on title and provode an emty alt unless an alt for navlogo as manually provided plz.

Reply:
Done, working on docs now.